frontend/src/layouts/Settings.tsx

```diff
@@ -93,6 +97,7 @@ function SettingsLayout(props: LayoutProps) {
                                 : ""
                             }
                             to="/admin/settings/topicarea"
+                            aria-current="page"
```

Comment:
Hi @denisquesada,
I think the aria-current="page" is only set for the current page link not for the others. Can you research the ARIA spec for side navigations to see what it says about it?

https://a11y-guidelines.orange.com/en/articles/using-aria-current-attribute/

Reply:
Good catch. I've updated a new revision. Many thanks.